### DISCUSSION TITLE
pre-releases and daily builds?

### DISCUSSION THREAD

**Discussion by prabhavmehra** [SixLabors/ImageSharp#2544] [Q&A] [answered]
Recently https://github.com/SixLabors/ImageSharp/pull/2543 got merged. I am unaware of the next release date where this will be included. are there any pre-releases,daily builds present where we can access ImageSharp with these arbitrary fixes? Whats the next best option? building the master branch and referencing that?

**Comment by prabhavmehra** [comment]
https://f.feedz.io/sixlabors/sixlabors/nuget/index.json isnt updated with the fix i need tho

**Reply by JimBobSquarePants** [reply, marked as answer]
Yes it is, do you have "Include prerelease" enabled in the package manager window?

<img  alt="image" src="https://github.com/SixLabors/ImageSharp/assets/385879/217f017b-08c2-42c0-b92f-ea66abdcc03e">


<img alt="image" src="https://github.com/SixLabors/ImageSharp/assets/385879/973304fc-8d6e-460d-9a58-49f9faac2c33">

**Reply by prabhavmehra** [reply]
@JimBobSquarePants  I am bit confused with feedz.io and how it works. I don't see the list you see on your side.

**Reply by prabhavmehra** [reply]
I am getting "Oops! a server error occured" when I try to add https://f.feedz.io/sixlabors/sixlabors/nuget/index.json to my repo url/import

**Reply by prabhavmehra** [reply]
Okay I got it working now. Looks like feedz.io is very buggy today.
